front-packages/akeneo-design-system/src/components/Input/MediaFileInput/MediaFileInput.tsx

```diff
@@ -272,9 +282,7 @@ const MediaFileInput = React.forwardRef<HTMLInputElement, MediaFileInputProps>(
           </>
         ) : null !== value ? (
           <>
-            <MediaFileImage
-              height={isCompact ? 47 : 120}
-              width={isCompact ? 47 : 120}
+            <PreviewImage
```

Comment:
The following code should do the work no ? Doing some tricky translation and using percentage seem to be overkill and dangerous
```suggestion
            <MediaFileImage
              height={isCompact ? 47 : 120}
              width={isCompact ? 47 : 120}
              fit="contain"
```

Reply:
with that solution it will fit in a square of 120*120, not sure that's what Felix had in mind. Dont see the danger in % since MediaFileInputContainer wich is the parent had it size define in absolute value.

Reply:
I think it's was Stephane want (mockup of Stephane https://www.notion.so/akeneo/File-3d6b2dfaee734e7cb1247aab485efd98).

By doing the %, now the file name is not correctly placed
Before:

![Screenshot from 2022-03-17 11-22-32](https://user-images.githubusercontent.com/7239572/158788959-3a757cbf-087e-4a0a-8768-31afc393d356.png)

Now:
![Screenshot from 2022-03-17 11-22-18](https://user-images.githubusercontent.com/7239572/158788953-e108a3fa-2120-4d90-9605-29ea208282cd.png)